python/pyarrow/parquet.py

```diff
@@ -310,6 +310,44 @@ def read_row_groups(self, row_groups, columns=None, use_threads=True,
                                            column_indices=column_indices,
                                            use_threads=use_threads)
 
+    def iter_batches(self, batch_size, row_groups=None, columns=None,
```

Comment:
In the other method `scan_contents`, batch_size is optional.

Reply:
Agreed. I added the same default value for `iter_batches()`